smpp/server.go

```diff
@@ -32,7 +32,7 @@ var (
 	msgIdCounter    int64 = 0
 )
 
-var ServerStubHandlers = map[pdu.ID]func(pdu.Body) pdu.Body{
+var HandlerRoutings = map[pdu.ID]func(pdu.Body) pdu.Body{
```

Comment:
Another global variable, not good. The semantics of net/http wrt ServeMux is better.